query-engine/connector-test-kit-rs/relation_link/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "relation_link"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+setup = { path = "../query-tests-setup", package = "query-tests-setup" }
```

Comment:
Was this crate only for experimenting, or is it part of the PR?

Reply:
Oh, good point I thought I excluded that. It was only for experimenting and fixing datamodel generator.